src/main/webapp/app/pool/pool-list/pool-list.component.html

```diff
@@ -13,8 +13,36 @@ <h1 class="col-md-12" jhiTranslate="pools.list.title">PCoIP pools</h1>
                 <th class="col-md-1"><p jhiTranslate="pools.list.headers.book">Book</p></th>
             </tr>
             </thead>
-
+            <jhi-alert></jhi-alert>
+            <jhi-alert-error></jhi-alert-error>
             <tbody>
+            <tr class="row">
+                <td class="col-md-3"><input type="text" placeholder="{{'pools.list.headers.name' | translate }}"
+                                            [(ngModel)]="displayName"></td>
+                <td class="col-md-1"><input type="text" style="width: 100%" placeholder="{{'pools.list.headers.min' | translate }}"
```

Comment:
Type should be "number"  also min constraint could be added.
Documentation: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number